### PR TITLE
Refs RHIROS-576 - fix conversation of GC env var

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: clowdapp.yaml [hunk 1]
```diff
@@ -130,7 +130,7 @@ parameters:
   value: "True"
 - description: The interval at which garbage collection should run (in seconds)
   name: GARBAGE_COLLECTION_INTERVAL
-  value: "86400"
+  value: '86400'
 - description: Number of days after which data is considered to be outdated
   name: DAYS_UNTIL_STALE
-  value: "45"
+  value: '45'
```
Comment (saltgen) [inline clowdapp.yaml:136]: @kgaikwad Are the single quotes necessary? I can see double quotes line # 130

**Review by saltgen — APPROVED** [review]
@kgaikwad Just a minor nitpick, rest looks good to me :+1:

**Review by patilsuraj767 — APPROVED** [review]
lgtm :+1:

File: clowdapp.yaml (hunk 1 above)
Comment (kgaikwad) [reply clowdapp.yaml]: Yes, not necessary. Reverted

**Comment by kgaikwad** [comment]
Thank you @saltgen & @patilsuraj767! 
Currently, ci.int jobs are failing with irrelevant errors so merging PR with job failure.

**Pull request merged**